src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/InviteUsers/Validation/SecretsManager/SecretsManagerSubscriptionUpdate.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Bit.Core.AdminConsole.Models.Business;
+using Bit.Core.AdminConsole.OrganizationFeatures.OrganizationUsers.InviteUsers.Models;
+using Bit.Core.AdminConsole.OrganizationFeatures.OrganizationUsers.InviteUsers.Validation.PasswordManager;
+using Bit.Core.Models.StaticStore;
+
+namespace Bit.Core.AdminConsole.OrganizationFeatures.OrganizationUsers.InviteUsers.Validation.SecretsManager;
+
+public class SecretsManagerSubscriptionUpdate
+{
+    public bool UseSecretsManger { get; }
+    public int? Seats { get; }
+    public int? MaxAutoScaleSeats { get; }
+    public int OccupiedSeats { get; }
+    public int AdditionalSeats { get; }
+    public int? PasswordManagerUpdatedSeatTotal { get; }
+    public Plan.SecretsManagerPlanFeatures SecretsManagerPlan { get; }
+    public int? AvailableSeats => Seats - OccupiedSeats;
+    public int SeatsRequiredToAdd => AdditionalSeats - AvailableSeats ?? 0;
+    public int? UpdatedSeatTotal => Seats + SeatsRequiredToAdd;
```

Comment:
Just in case we do end up using this (contra my comment above), note that this still has a bunch of un-updated PM references (rather than SM).

Reply:
All the Seats stuff relates to the SM seat count separate from Password manager.